src/Mobs/Monster.cpp

```diff
@@ -14,13 +14,15 @@
 #include "../MonsterConfig.h"
 #include "../BoundingBox.h"
 
+#include "Items/ItemSpawnEgg.h"
+
 #include "../Chunk.h"
 #include "../FastRandom.h"
 
 #include "PathFinder.h"
 #include "../Entities/LeashKnot.h"
 
-
+const int TPS = 20;
```

Comment:
This should actually go in one of the global headers, I'm rather surprised we don't have such a constant already, but Monster.cpp isn't really the right place for it.

Reply:
```Defines.h``` maybe ?